test/integration/dialects/postgres/query-generator.test.js

```diff
@@ -870,7 +873,7 @@ if (dialect.match(/^postgres/)) {
           expectation: 'CREATE INDEX \"bar\" ON \"User\" (\"username\", \"isAdmin\")'
         }, {
           arguments: ['mySchema.User', ['username', 'isAdmin'], {}, 'User'],
```

Comment:
True .. (this is actually the same discussion as we are having below, right :D ?)

Then perhaps if tablename = string, use `quoteIdentifiers` else `quoteTable`?

(Sorry for being so indecisive @mikrofusion - but we need to make sure we still support legacy usecases + migrations)